src/site/content/en/react/quicklink-react-spa/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: Speed up navigations in React with quicklink
+subhead: |
+  Automatically prefetch in-viewport links with quicklink for React single page applications.
+date: 2020-06-08
+hero: hero.png
+description: |
+  quicklink is a library to achieve faster subsequent page-loads by prefetching in-viewport links during idle time.
+authors:
+  - addyosmani
+  - demianrenzulli
+---
+
+[Prefetching](https://web.dev/link-prefetch/) is a technique to speed up navigations by downloading the resources for the next page, ahead of time. [Quicklink](https://github.com/GoogleChromeLabs/quicklink) is a library that allows you to implement this technique at scale, by automatically prefetching links as they come into the view.
+
+In multi page apps the library prefetches documents (e.g. `/article.html`), for in-viewport links, so that when the user clicks on these links they can be picked up from the [HTTP cache](https://web.dev/http-cache/).
```

Comment:
```suggestion
In multi-page apps the library prefetches documents (e.g. `/article.html`), for in-viewport links, so that when the user clicks on these links they can be picked up from the [HTTP cache](https://web.dev/http-cache/).
```

Reply:
Please update `multi page` to `multi-page` throughout the article